README.md

```diff
@@ -3,29 +3,21 @@
 [![Build status](https://ci.woodpecker-ci.org/api/badges/woodpecker-ci/plugin-surge-preview/status.svg)](https://ci.woodpecker-ci.org/woodpecker-ci/plugin-surge-preview)
 [![Docker Image Version (latest by date)](https://img.shields.io/docker/v/woodpeckerci/plugin-surge-preview?label=DockerHub%20latest%20version&sort=semver)](https://hub.docker.com/r/woodpeckerci/plugin-surge-preview/tags)
 
-Woodpecker plugin to deploy static pages for reviewing to [surge.sh](https://surge.sh/).
-
-## Build
-
-Build the Docker image with the following command:
-
-```sh
-docker buildx build -t woodpeckerci/plugin-surge-preview . --load
-```
```

Comment:
I would like to keep the build instructions.

Reply:
Having them in the README as the first block suggests that they might be needed to use the plugin (for some unexperienced users). How about turning it into a makefile rule?

Reply:
I don't really think that's a real concern, but in that case you can just add some hints like `For local development, build the container ...`

Reply:
Just swap `Usage` and `Build`

Reply:
I just had this case a few days ago (for a different plugin). 

The Q is: experienced devs know how to do it anyway or will find the Makefile rule. For unexperienced ones, it will cause such questions/confusion. 

Anyhow, added it back.